virtwho/migrate/migrateconfiguration.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+"""
+For moving variables from system environment to general config file
+
+Copyright (C) 2020 William Poteat <wpoteat@redhat.com>
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+
+import os
+
+SYSCONFIG_FILENAME = '/etc/sysconfig/virt-who'
+GENERAL_CONFIG_FILENAME = '/etc/virt-who.conf'
+
+def main():
+    migrate_env_to_config()
+
+def migrate_env_to_config(sysconfig_filename=None, general_config_filename=None):
+    if not sysconfig_filename:
+        sysconfig_filename = SYSCONFIG_FILENAME
+    if not general_config_filename:
+        general_config_filename = GENERAL_CONFIG_FILENAME
+
+    interval = None
+    debug = None
+    one_shot = None
+
+    # read know env variables
+    if os.environ.get('VIRTWHO_INTERVAL'):
+        interval = os.environment['VIRTWHO_INTERVAL']
+    if os.environ.get('VIRTWHO_DEBUG'):
+        debug = os.environ['VIRTWHO_DEBUG']
+    if os.environ.get('VIRTWHO_ONE_SHOT'):
+        one_shot = os.environ['VIRTWHO_ONE_SHOT']
+
```

Comment:
In general I would use for getting environment variables (`interval`, `debug` and `one_shot`) something like this:
```python
interval = os.environ.get('VIRTWHO_INTERVAL', None)
```